client/sttp-client/src/main/scala/sttp/tapir/client/sttp/SttpClientInterpreter.scala

```diff
@@ -1,10 +1,19 @@
 package sttp.tapir.client.sttp
 
-import sttp.client3.Request
+import sttp.client3.{Request, SttpBackend}
 import sttp.model.Uri
+import sttp.monad.MonadError
 import sttp.tapir.{DecodeResult, Endpoint}
 
-trait SttpClientInterpreter {
+trait SttpClientInterpreter extends SttpClientInterpreterPlatformSpecific {
+
+  def toClientUnsafe[F[_], I, E, O](e: Endpoint[I, E, O, Any], baseUri: Uri, backend: SttpBackend[F, Any])(implicit
```

Comment:
👍 

We should also probably have a safe variant, which returns the result wrapped in a `DecodeError`